lib/Conf.ml

```diff
@@ -185,24 +185,29 @@ let ocaml_version_conv =
 module Formatting = struct
   let section = `Formatting
 
+  let no_align = "Vertical alignment is no longer a promoted behavior."
```

Comment:
I think this would sound better:
```suggestion
  let no_align = "Vertical alignment is discouraged."
```

Reply:
I'm not a big fan of discouraged, what do you think of inadvisable?

Reply:
Is it necessary to add any such sentence? It seems enough to state that it is deprecated and will be removed by version 1.0.

Reply:
indeed, fixed!